src/transformers/pipelines/audio_utils.py

```diff
@@ -70,6 +75,10 @@ def ffmpeg_microphone(
             The indentifier of the input device to be used by ffmpeg (i.e. ffmpeg's '-i' argument). If unset,
             the default input device will be used. See `https://www.ffmpeg.org/ffmpeg-devices.html#Input-Devices`
             for how to specify and list input devices.
+        ffmpeg_additional_args (`list`, *optional*):
+            Additional arguments to pass to ffmpeg, can include arguments like -nostdin for running as a background
+            process.
```

Comment:
It would be good to show a specific example here to show the user how to pass in the arguments e.g. `["-nostdin"]`. How should the user pass in if the arguments are a flag then a series of options e.g. `ffmpeg -flag argument_1 argument_2`?